src/mqtt_t.cpp

```diff
@@ -68,18 +76,19 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) ;
 void initMqtt()
 {
   mMqttMutex = xSemaphoreCreateMutex();
-  
   smMqttConnectState=SM_MQTT_DISCONNECTED;
   smMqttConnectStateOld=SM_MQTT_DISCONNECTED;
   u8_mWaitConnectCounter=0;
-  
+  PLbuf.reserve(512);
```

Comment:
There are several buffer allocations of size 512. If they are somehow related, I would suggest to use a common definition. 
Example: 
`constexpr std::size_t MQTT_DATA_BUFFER_SIZE = 512`

and use it in all of the related buffer allocations.

```
PLbuf.reserve(MQTT_DATA_BUFFER_SIZE);
...
mqttClient.setBufferSize(MQTT_DATA_BUFFER_SIZE);
...
char buf[MQTT_DATA_BUFFER_SIZE];
```